custom-vu/src/main/kotlin/jces1209/vu/page/boards/view/MovingIssue.kt

```diff
@@ -43,19 +43,34 @@ class MovingIssue(
             .moveToElement(driver
                 .findElements(moveZoneLocator)
                 .first { it.isDisplayed })
-            .release()
             .perform()
 
+        Thread.sleep(1000)

```

Comment:
Please add a comment - `why is it necessary to add the sleep`

Note: for other sleeps too

Reply:
Added comments